src/lib/profiles/security/WeaveSecurity.h

```diff
@@ -243,6 +252,45 @@ enum
     kTag_GroupKeySignature_KeyId                = 2,    //  [ unsigned int ] Weave KeyId to be used to generate and verify the signature
     kTag_GroupKeySignature_Signature            = 3,    //  [ byte string ] Signature bytes themselves.
 
+    // ---- Context-specific Tags for GetCertificateRequest Message Structure ----
+    kTag_GetCertReqMsg_ReqType                  = 1,    // [ unsigned int ] Identifies the certificate request type.
+    kTag_GetCertReqMsg_Authorize_PairingToken   = 2,    // [ byte string, optional ] Pairing token from the service, in BER integer encoding.
+                                                        //                           This is used to authorize GetCertificate to the service.
+    kTag_GetCertReqMsg_Authorize_PairingInitData= 3,    // [ byte string, optional ] Pairing initialization data from the service, in BER integer encoding.
+                                                        //                           This is used to authorize GetCertificate to the service.
+    kTag_GetCertReqMsg_OpDeviceCert             = 4,    // [ structure ] Weave operational device certificate.
+                                                        //               This has the same internal structure as an kTag_WeaveCertificate.
+    kTag_GetCertReqMsg_OpRelatedCerts           = 5,    // [ array, optional ] An optional array of intermediate Weave certificates, which are needed to
+                                                        //                     validate Weave operational device certificate. May be omitted if validators
+                                                        //                     are expected to have the necessary certificates for validation.
+    kTag_GetCertReqMsg_MfrAttest_WeaveCert      = 6,    // [ structure ] Weave device certificate provisioned at the factory at manufacturing time.
+                                                        //               This has the same internal structure as an kTag_WeaveCertificate.
+    kTag_GetCertReqMsg_MfrAttest_WeaveRelCerts  = 7,    // [ array, optional ] An optional array of intermediate Weave certificates, which are needed to
+                                                        //                     validate included Weave device certificate. May be omitted if validators
+                                                        //                     are expected to have the necessary certificates for validation.
+    kTag_GetCertReqMsg_MfrAttest_X509Cert       = 8,    // [ byte string ] X509 device certificate provisioned at the factory at manufacturing time.
+                                                        //                 This is an ASN1 encoded ECDSA or RSA certificate TLV-encoded as data blob.
+    kTag_GetCertReqMsg_MfrAttest_X509RelCerts   = 9,    // [ array, optional ] An optional array of intermediate X509 certificates, which are needed to
```

Comment:
The comment should describe the type of the array's elements.

Reply:
updated